lib/shared/permissions.js

```diff
@@ -73,6 +73,21 @@ exports.isElevated = () => {
 };
 
 /**
+ * @summary Quote a string
+ * @function
+ * @private
+ *
+ * @param {String} string - input string
+ * @returns {String} quoted string
+ *
+ * @example
+ * const result = doubleQuoteString('foo');
+ */
+const doubleQuoteString = (string) => {
+  return `"${string}"`;
```

Comment:
I'll test this changes on Linux and macOS first. At least in Windows, un-escaped nested quotes are what makes everything work :P